### PR TITLE
ceph-detect-init: FreeBSD introduction of bsdrc

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by wjwithagen** [comment]
@tchaikov 
I think it would... 
From what I see you went for the mock.patch.multiple() version.
Also interresting to see that you have mocked the actual definitions of routines, looks like a cleaner approach.

Will run it thru some testing.

**Comment by wjwithagen** [comment]
@tchaikov 
How can I easilly integrate this PR into my working tree... 
Usually I just open and copy the files, but git would be able to do that in a easier way.
I've seen something like cherrypick??

**Comment by tchaikov** [comment]
i'd fetch this PR using
```
git fetch -f origin pull/10748/head:wip-10748
```
and start working on "wip-10748".

**Comment by tchaikov** [comment]
i'd fetch this PR using
```
git fetch -f origin pull/11829/head:wip-11829
```
and start working on "wip-11829".

**Comment by wjwithagen** [comment]
@tchaikov 
Oke thanx.

**Comment by tchaikov** [comment]
@dachary could you take a look?

**Comment by dachary** [comment]
jenkins test this please (general jenkins failure)

File: src/ceph-detect-init/ceph_detect_init/__init__.py [hunk 1]
```diff
@@ -90,11 +92,19 @@ def _normalized_distro_name(distro):
...
+    if platform.system() == 'Linux':
+        linux_distro = platform.linux_distribution(
+            supported_dists=platform._supported_dists + ('alpine',))
+        logging.debug('platform_information: linux_distribution = ' +
+                      str(linux_distro))
+        distro, release, codename = linux_distro
+    elif platform.system() == 'FreeBSD':
+        distro = 'freebsd'
+        release = platform.release()
+        codename = platform.version().split(' ')[3].split(':')[0]
```
Comment (tchaikov) [inline src/ceph-detect-init/ceph_detect_init/__init__.py]: done

**Pull request merged**